ui/src/components/AuthConsent/__snapshots__/index.test.jsx.snap

```diff
@@ -1,54 +1,334 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should render AuthConsent page 1`] = `
-<Router
-  history={
-    Object {
-      "action": "POP",
-      "block": [Function],
-      "canGo": [Function],
-      "createHref": [Function],
-      "entries": Array [
-        Object {
-          "hash": "",
-          "pathname": "/",
-          "search": "",
-          "state": undefined,
-        },
-      ],
-      "go": [Function],
-      "goBack": [Function],
-      "goForward": [Function],
-      "index": 0,
-      "length": 1,
-      "listen": [Function],
-      "location": Object {
-        "hash": "",
-        "pathname": "/",
-        "search": "",
-        "state": undefined,
-      },
-      "push": [Function],
-      "replace": [Function],
-    }
-  }
->
-  <withRouter(WithStyles(AuthConsent))
-    clientId="clientId"
-    formData={
-      Object {
-        "description": "description",
-        "expires": 2022-02-02T12:00:00.000Z,
-        "scopes": Array [
-          "scopes",
-        ],
-      }
-    }
-    onExpirationChange={[MockFunction]}
-    onInputChange={[MockFunction]}
-    onScopesChange={[MockFunction]}
-    registeredClientId="registeredClientId"
-    transactionID="transactionID"
-  />
-</Router>
+<DocumentFragment>
```

Comment:
Turned out enzyme snapshots / serializing was useless. Switched to react-testing-library to have the actual html tree here